onelogin_rules/onelogin_active_login_activity.py

```diff
@@ -27,14 +27,16 @@ def rule(event):
     # First, keep a list of unique user ids that have logged in from this ip address
     event_key = get_key(event)
     user_ids = get_string_set(event_key)
+    # the user id of the user that has just logged in
+    user_id = str(event.get('user_id'))
```

Comment:
I decided to put this here rather than in the `put_string_set` and `add_to_string_set` methods since those methods explicitly define str input.  I'd worry about people passing an object or something where the str() conversion in those methods might return something 'unexpected'?

Reply:
I agree, I would rather have an error (so that at least I know something is wrong) versus something that runs without issue but isn't actually doing what I think it does.